helpers/pack_source_audioflingerglue-localbuild.sh

```diff
@@ -12,15 +14,15 @@ fold=hybris/mw/$pkg
 rm -rf $fold
 mkdir $fold
 
-mkdir -p $fold/out/target/product/${DEVICE}/system/${DROIDLIB}
-mkdir -p $fold/out/target/product/${DEVICE}/system/bin
+mkdir -p $fold/out/target/product/${OUT_DEVICE}/system/${DROIDLIB}
+mkdir -p $fold/out/target/product/${OUT_DEVICE}/system/bin
 mkdir -p $fold/external/audioflingerglue
 
 cp ./external/audioflingerglue/*.h $fold/external/audioflingerglue/
 cp ./external/audioflingerglue/hybris.c.in $fold/external/audioflingerglue/
 # Remove audioflingerglue bits from out/ (otherwise it would cause a conflict within droid-hal-$DEVICE):
```

Comment:
Small nitpick on here should be $OUT_DEVICE too for consistency, even when this commit will be short-lived :) (due to what's coming in the next PR of yours)
I'll push to your branch and merge (will also reword the commit message a little)

Reply:
Do you mean `$OUT_DEVICE` instead of `${OUT_DEVICE}`?

Reply:
I meant in the comment: `droid-hal-$DEVICE` -> `droid-hal-$OUT_DEVICE` is semantically correct. No worries, I've already fixed that for ya

Reply:
{curly brackets} is just one of the many mnemonics, I kept it consistent with history